### ISSUE TITLE
Pyne is deprecated as of python 3.7

### ISSUE THREAD

**Issue by ZoeRichter** [pyne/pyne#1232]
I have python version 3.7.4 installed on my machine running Ubuntu 18.04.  When I import pyne, I get the following block of warnings:
```
/home/zoe/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  """
/home/zoe/anaconda3/lib/python3.7/importlib/_bootstrap.py:219: QAWarning: pyne.data is not yet QA compliant.
  return f(*args, **kwds)
/home/zoe/anaconda3/lib/python3.7/importlib/_bootstrap.py:219: QAWarning: pyne.material is not yet QA compliant.
  return f(*args, **kwds)
/home/zoe/.local/lib/python3.7/site-packages/pyne/serpent.py:11: QAWarning: pyne.serpent is not yet QA compliant.
  warn(__name__ + " is not yet QA compliant.", QAWarning)

```

If it will stop working as of 3.8, I'm thinking that there should be something to change the ABC implementation for users with python 3.7+

**Comment by gonuke** [comment]
Thanks for pointing this out @ZoeRichter - I have a fix in the works in PR #1233

**Comment by ZoeRichter** [comment]
It looks like #1233 has been merged, so I'll close this issue.
